packages/cli/src/util/dev/server.ts

```diff
@@ -818,6 +841,8 @@ export default class DevServer {
         env['VERCEL_URL'] = this.address.host;
       } else if (name === 'VERCEL_REGION') {
         env['VERCEL_REGION'] = 'dev1';
+      } else if (name === 'VERCEL_ENV') {
+        env['VERCEL_ENV'] = 'development';
```

Comment:
Is this `injectSystemValuesInDotenv()` function still necessary? Seems like it will get overridden by the code you added above.

Reply:
deleted the use of `injectSystemValuesInDotenv()` entirely